src/activities/drawnumber/drawnumber.js

```diff
@@ -137,50 +240,93 @@ function drawSegment(pointIndex) {
         }
 
         if (pointIndex == items.pointImageRepeater.count-1) {
-            for (var i = 1; i < dataset[currentLevel].coordinates.length; i++) {
-                items.segmentsRepeater.itemAt(i-1).opacity = 0
+
+            if(mode == "drawletters"){
```

Comment:
if you only have one dataset, you won't have to do 3 times the same code below ;)